### ISSUE TITLE
crash in scylla 

### ISSUE THREAD

**Issue by slivne** [scylladb/scylla#3018]
```
scylla/seastar/core/iostream.hh:210: output_stream<CharType>::~output_stream() [with CharType = char]: Assertion `!_in_batch' failed.
Aborting on shard 0.
```

**Comment by slivne** [comment]
fixed by scylladb/seastar@35095449e41c1062415a02862c41cc8501788bc0

**Comment by avikivity** [comment]
Please use more descriptive titles.
